x/oracle/keeper/hooks_test.go

```diff
@@ -9,6 +9,12 @@ func (s *IntegrationTestSuite) TestHooks_AfterTokenRegistered() {
 	h := s.app.OracleKeeper.Hooks()
 	s.Require().Len(s.app.OracleKeeper.AcceptList(s.ctx), 1)
 
+	// TODO: These hooks need to respond to the Token.Blacklist field
+	// (as well as update symbol denom and exponent if they ever change).
+	// Blacklisting in particular should actually eliminate the base
+	// denom from the oracle (and un-blacklisting, for whatever reason,
+	// should do the opposite.)
+
 	// require that an existing token does not change the accept list
 	h.AfterTokenRegistered(s.ctx, leveragetypes.Token{
```

Comment:
@adamewozniak Could you add the necessary oracle changes to this PR after reviewing?

Reply:
@toteki I don't believe this update is necessary; the oracle module should be relatively agnostic of the x/leverage module's functionality. 

For example, if we used the oracle as a service for other blockchains over IBC, we would not want a leverage "blacklist" to remove those from the oracle's params.

In the case where we **definitely do** want to remove a denom from the oracle, a parameter governance proposal is still possible.

Reply:
@toteki I don't believe this update is necessary; the oracle module should be relatively agnostic of the x/leverage module's functionality. 

For example, if we used the oracle as a service for other blockchains over IBC, we would not want a leverage "blacklist" to remove those from the oracle's params.

In the case where we **definitely do** want to remove a denom from the oracle, a parameter governance proposal is still possible.

Reply:
Ok - in that case, we can remove the TODO for removing oracle accepted assets as a result of leverage module blacklisting.

Would it be ok to make a separate umee repo issue to cover "update of SymbolDenom / Exponent" situation that was mentioned?

Reply:
Imo that's a similar scenario, they could just be updated via param change proposals